ReportPortal.Tests/ReportPortal.E2E.Core/Utility/ApiClient.cs

```diff
@@ -0,0 +1,21 @@
+﻿using ReportPortal.E2E.Core.Extensions;
+using ReportPortal.E2E.Core.HttpMessageHandlers;
+using ReportPortal.E2E.Core.HttpMessageHandlers.HttpClient;
+using HttpClient = ReportPortal.E2E.Core.HttpMessageHandlers.HttpClient.HttpClient;
+
+namespace ReportPortal.E2E.Core.Utility
+{
+    public static class ApiClient
+    {
+        public static IHttpClient Get(string baseUrl, AuthorizationMessageHandler authorizationMessageHandler = null)
+        {
+            var clientName = TestsBootstrap.Instance.Configuration.GetSection("ApiClient").GetValueOrThrow();
+            return clientName switch
+            {
+                "HttpClient" => authorizationMessageHandler == null ? new HttpClient(baseUrl) : new HttpClient(baseUrl, authorizationMessageHandler),
```

Comment:
As a recommendation better also create enum for such consts.
The same as for browsers name, sorry forget about this.

Reply:
Agree. Done